lib/sidekiq/cli.rb

```diff
@@ -269,6 +269,19 @@ def validate!
         logger.info @parser
         die(1)
       end
+
+      errors = []
+      [:concurrency, :index, :timeout].each do |arg|
+        if options.has_key? arg
+          Integer(options[arg]) rescue errors.push("#{arg.to_s} must be an integer")
+        end
+      end
+      if options.has_key? :verbose
+        val = options[:verbose]
+        errors.push("verbose must be true or false") unless val == true || val == false
```

Comment:
verbose is a flag given to us by optparse.  There's no need to validate it at all.